src/test/java/openfasttrack/importer/legacytag/TestPathConfig.java

```diff
@@ -0,0 +1,107 @@
+package openfasttrack.importer.legacytag;
+
+/*-
+ * #%L
+ * OpenFastTrack
+ * %%
+ * Copyright (C) 2016 - 2018 hamstercommunity
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * #L%
+ */
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+import java.nio.file.Paths;
+
+import org.junit.Test;
+
+public class TestPathConfig
+{
+    @Test
+    public void testGlobMatches()
+    {
+        assertMatches("glob:pattern", "pattern", true);
+    }
+
+    @Test
+    public void testGlobUsedByDefault()
+    {
+        assertMatches("**/pattern", "blah/blubb/pattern", true);
+    }
+
+    @Test
+    public void testGlobWithStarStarMatches()
+    {
+        assertMatches("glob:**/pattern", "blah/blubb/pattern", true);
+        assertMatches("glob:**/pattern", "/pattern", true);
+        assertMatches("glob:**/pattern", "pattern", false);
+        assertMatches("glob:**/pattern", "attern", false);
+    }
+
+    @Test
+    public void testGlobDoesntMatches()
+    {
+        assertMatches("glob:pattern", "wrongpattern", false);
+    }
+
+    @Test
+    public void testRegexpMatches()
```

Comment:
Rename to "testLiteralMatches()" to avoid confusion.

Reply:
:white_check_mark: